pom.xml

```diff
@@ -37,7 +37,7 @@
         <jstl.version>1.1.2</jstl.version>
         <jta.version>1.1</jta.version>
         <poi.version>3.9</poi.version>
-        <resource-server.version>1.0.46</resource-server.version>
+        <resource-server.version>1.0.47-SNAPSHOT</resource-server.version>
```

Comment:
We'll need to wait for an official release of resource server to be cut before this is merged.